app/screens/settings-screen/notifications-screen.tsx

```diff
@@ -0,0 +1,410 @@
+import * as React from "react"
+import { useI18nContext } from "@app/i18n/i18n-react"
+
+import { View } from "react-native"
+import { makeStyles, Switch, Text } from "@rneui/themed"
+
+import { Screen } from "../../components/screen"
+
+import { gql } from "@apollo/client"
+import {
+  AccountDisableNotificationCategoryMutation,
+  AccountDisableNotificationChannelMutation,
+  AccountEnableNotificationCategoryMutation,
+  AccountEnableNotificationChannelMutation,
+  NotificationChannel,
+  NotificationSettings,
+  useAccountDisableNotificationCategoryMutation,
+  useAccountDisableNotificationChannelMutation,
+  useAccountEnableNotificationCategoryMutation,
+  useAccountEnableNotificationChannelMutation,
+  useNotificationSettingsQuery,
+} from "@app/graphql/generated"
+import { useIsAuthed } from "@app/graphql/is-authed-context"
+
+const useStyles = makeStyles(() => ({
```

Comment:
our pattern has been to put that style definition at the end of the file

Reply:
Moved it